users/variables.tf

```diff
@@ -5,8 +5,8 @@
 # ------------------------------------------------------------------------------
 
 variable "godlike_usernames" {
-  type        = map(string)
-  description = "The usernames associated with the god-like accounts to be created, which are allowed to access the terraform backend, are IAM administrators for the Users account, and are allowed to assume any role that has a trust relationship with the Users account.  The format first.last is recommended.  The keys are the usernames and the values are empty strings (since they are not presently used). Example: { \"firstname1.lastname1\" = \"\",  \"firstname2.lastname2\" = \"\" }"
+  type        = list(string)
+  description = "The usernames associated with the god-like accounts to be created, which are allowed to access the terraform backend, are IAM administrators for the Users account, and are allowed to assume any role that has a trust relationship with the Users account.  The format first.last is recommended.  Example: [ \"firstname1.lastname1\",  \"firstname2.lastname2\"]"
```

Comment:
Remove stray space after "[".

Reply:
Thanks - fixed in https://github.com/cisagov/cool-accounts/pull/29/commits/316f31c64682c8d52e1c87633266e686fb628988.